docs/resources.md

```diff
@@ -0,0 +1,39 @@
+# Resources
+
+The following are resources about Melange collected around the web.
+
+## Blog posts, podcasts, talks
```

Comment:
Maybe we can add this one by @davesnx 

https://sancho.dev/blog/whats-possible-with-melange

There is also another old Ahrefs blog post from 2021: 

https://tech.ahrefs.com/building-ahrefs-codebase-with-melange-9f881f6d022b

And these two:

https://dev.to/psb/getting-started-with-reasonreact-and-melange-13hd
https://dev.to/psb/reason-and-react-meta-frameworks-d7n

Reply:
Added suggested blog posts and removed official Melange blog posts in https://github.com/melange-re/melange-re.github.io/pull/189/commits/a137ca1b37653d8b39377878bf52f5908d2ac384